jnosql-dynamodb/pom.xml

```diff
@@ -57,6 +57,12 @@
             <artifactId>dynamodb-enhanced</artifactId>
             <version>${dynamodb.version}</version>
         </dependency>
+        <dependency>
```

Comment:
Hey Max, how about including this as JNoSQL parent?
This might be valuable to everyone.

Reply:
@otaviojava, sorry, I didn't understand... I'm still working on this PR... do you think that's a good idea to merge this PR as it is now into the main branch?

Reply:
Oh... I think that I understand now, @otaviojava !!! You mean to add the DataFaker at JNoSQL parent POM!!! When I saw it on my smartphone, I didn't realize you were talking about it... LOL

Reply:
I'm gonna add this one to the JNoSQL parent POM!

Reply:
It's done!: https://github.com/eclipse/jnosql/pull/473